fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1436,6 +1435,20 @@ public void removeExpiredAndTimeoutTxns(long currentMillis) {
         }
     }
 
+    public void abortTimeoutTxns(long currentMillis) {
```

Comment:
`removeExpiredAndTimeoutTxns` function can be removed.

Reply:
done